### PR TITLE
Removed unexpected bracket on line 64, Added Terror Pound, adjusted P…

### PR DESCRIPTION
**Pull request by Xrez31**
Removed unexpected bracket on line 64, Added Terror Pound, adjusted Pound Price

### CODE REVIEW

File: config.lua [hunk 1]
```diff
@@ -173,6 +172,13 @@ Config.BoatPounds = {
 		Marker = vector3(-738.67, -1400.43, 4.0),
 		Spawner = vector3(-738.33, -1381.51, 0.12),
 		Heading = 137.85
+	},
+
+	Terrorist_Dock = {
+        Marker = vector3(3130.80, 2180.00, 4.40),
+        Spawner = vector3(3112.60, 2173.00, -0.50),
+        Heading = 121.02
+		Hidden = true
```
Comment (alvinveroy) [inline config.lua:183]: Hidden Config value is not implemented on 
Boat Garages in client/main.lua 

https://github.com/lockdownstudio/esx_advancedgarage/blob/270bed0fb26cb858e654720ae5c12ab363245283/client/main.lua#L1684

Check Line 1718 on how to implement Hidden value on  Boat Garages.

https://github.com/lockdownstudio/esx_advancedgarage/blob/270bed0fb26cb858e654720ae5c12ab363245283/client/main.lua#L1718

**Review by alvinveroy — CHANGES_REQUESTED** [review]
Please change client/main.lua to implement Hidden value on boat garages.

File: client/main.lua [hunk 2]
```diff
@@ -1683,6 +1683,7 @@ function CreateBlips()
```
Comment (alvinveroy) [inline client/main.lua:1685]: Hidden value declaration on config without implementation in client/main.lua

https://github.com/lockdownstudio/esx_advancedgarage/blob/270bed0fb26cb858e654720ae5c12ab363245283/config.lua#L88

https://github.com/lockdownstudio/esx_advancedgarage/blob/270bed0fb26cb858e654720ae5c12ab363245283/config.lua#L129

https://github.com/lockdownstudio/esx_advancedgarage/blob/270bed0fb26cb858e654720ae5c12ab363245283/config.lua#L328

Either delete the hidden value or create an implementation on client/main.lua

**Review by alvinveroy — CHANGES_REQUESTED** [review]
Please double check the config.lua, several implementations of the key Hidden are not in client/main.lua
either remove it from the config or create the implementation for each logic statements.

File: client/main.lua [hunk 3]
```diff
@@ -1864,6 +1875,28 @@ function RefreshJobBlips()
...
+				local blip = AddBlipForCoord(v.Marker)
+
+				SetBlipSprite (blip, Config.JPoundBlip.Sprite)
+				SetBlipColour (blip, Config.JPoundBlip.Color)
+				SetBlipDisplay(blip, Config.JPoundBlip.Display)
+				SetBlipScale  (blip, Config.JPoundBlip.Scale)
+				SetBlipAsShortRange(blip, true)
+
+				BeginTextCommandSetBlipName("STRING")
+				AddTextComponentString(_U('blip_mechanic_pound'))
```
Comment (alvinveroy) [inline client/main.lua:1896]: Missing value in locales/en.lua

**Review by alvinveroy — CHANGES_REQUESTED** [review]
Add the missing value in locales/en.lua

File: client/main.lua (hunk 3 above)
Comment (Xrez31) [reply client/main.lua:1896]: image

Yes po meron na hehe.

**Pull request merged**